src/components/List/List.js

```diff
@@ -64,6 +68,19 @@ export default class List extends React.Component {
     </FlexView>
   );
 
+  templateError = (error) =>(
```

Comment:
note that you're passing `{ error }` below (good, as per convention every "`template`" methods receives a "`locals`" object), but here you're not destructuring, leading to `error.error` accesses below (also see other comment below)